### ISSUE TITLE
GestureDetector groups / Listener group / GestureDetector links

### ISSUE THREAD

**Issue by subzero911** [flutter/flutter#95915]
## The problem

I encountered a situation where I want to listen for pointerDown event on one widget and catch pointerUp on another widget.
But it's impossible. If I started tap on GestureDetector 1, I cannot end it on GestureDetector 2 (it will be considered like GestureDetector 1 tapCancel event). The same to the Listener widget. If I touched the screen on Listener 1, then Listener 2 knows nothing about a Listener 1`s pointer and will not catch the pointerUp event.

## Use case

For example, I'm crafting a WeChat clone and want to send a voice message.
I press the button and hold it down, waiting for modal screen (overlay) to appear. It has 3 buttons: "send", "translate" and "cancel". And I want to release the tap on one of them.
It looks like a basic task, but I cannot find any ready-made solution for it. I'm thinking about workarounds like listening raw pointer data from SingletonFlutterWindow and using MouseRegion to check if I'm inside the certain widget and the pointer is down. But I want some simple solution out-of-the box.

## Proposal

My idea is to introduce GestureDetectors groups.
You could connect GestureDetectors by tag. Say, if two or more GestureDetectors have the same tag, they are considered to be in the same group, and you can start the gesture on the one and to end it on the another. The same to Listeners.

**Comment by bernaferrari** [comment]
Doesn't DragTarget help you?

**Comment by subzero911** [comment]
> Doesn't DragTarget help you?

How exactly it will help?
I should have some Draggable widget for it. But there's nothing to drag.
Any help is appreciated

**Comment by darshankawar** [comment]
@subzero911 
Can you provide us a code sample representing your use case along with actual vs expected result, so that we can take a look and verify it properly ?
Thanks.

**Comment by subzero911** [comment]
> Doesn't DragTarget help you?

I tried DragTarget and it worked like charm.
I used LongPressDraggable with hapticFeedback on a button to simulate longpress (button has empty handler and is just a decoration). I also used an empty SizedBox as a `feedback` widget.
In DragTarget, `onWillAccept` is working as "onEnter" and `onLeave` is "onExit", and `onAccept` to perform some actions. What's enough for my tasks.
